src/components/teampage/team-members/team-member-styles.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components"
+import { size } from "../../../constants/devices"
+
+const CardPosition = styled.div`
+  min-width: calc(50% - 30px);
+  height: 300px;
+  display: inline-flex;
+  margin-bottom: ${props =>
+    props.isLast && props.index % 2 === 0 ? "0" : "8%"};
+  margin-left: ${props => (props.index % 2 === 0 ? "0" : "60px")};
+  position: relative;
+
+  @media only screen and (max-width: ${size.tablet}px) {
+    display: flex;
+    min-width: 100%;
+    margin: 0;
+    height: initial;
+  }
+`
+const ContactCardContainer = styled.div`
+  display: inline-flex;
+  min-width: 100%;
+  position: absolute;
+  top: ${props => (props.index % 2 === 0 ? "0" : "180px")};
+  left: ${props => (props.position === "Executive Director" ? "25%" : 0)};
```

Comment:
I'm not sure about this – if position ever changes it will break it. Can you find a way to not use a content-based approach here?